python/tests/api/logger/experimental/multi_dataset_logger/test_multi_dataset_rolling_logger.py

```diff
@@ -68,6 +78,35 @@ def test_happy_path() -> None:
     assert writer.write_count == 1
 
 
+def test_no_segments_schema() -> None:
+    writer = TestWriter()
+
+    # Make a fancy embedding schema with no segmentation
+    l: List[List[float]] = [[1.0 for i in range(0, 50)] for i in range(0, 30)]
```

Comment:
can we use a longer name there, single lower case L is hard to distinguish from digit one.